django-verdant/rca/models.py

```diff
@@ -3381,6 +3407,7 @@ class StaffPage(Page, SocialFields):
         'collaborations',
         'publications_exhibitions',
         'ad_username',
+        'supervised_students'
```

Comment:
Do this need adding as a ('field_name', SerializerClass) tuple to give you the representation you want?